### ISSUE TITLE
Verify crashes if no directory is specified

### ISSUE THREAD

**Issue by rchande** [dotnet/try#310]
The command line handling should probably inform you of the necessary argument instead of crashing.

```
C:\MLS-LS>dotnet try verify
Unhandled exception: System.Reflection.TargetInvocationException: Exception has been thrown by the target of an invocation. ---> System.NullReferenceException: Object reference not set to an instance of an object.
   at WorkspaceServer.PackageRegistry.CreateForTryMode(DirectoryInfo project, PackageSource addSource)
   at MLS.Agent.CommandLine.CommandLineParser.<>c.<Create>b__7_2(VerifyOptions verifyOptions, IConsole console, StartupOptions startupOptions) in F:\workspace.13\_work\1\s\MLS.Agent\CommandLine\CommandLineParser.cs:line 91
   at MLS.Agent.CommandLine.CommandLineParser.<>c__DisplayClass7_0.<Create>b__33(VerifyOptions options, IConsole console, StartupOptions startupOptions) in F:\workspace.13\_work\1\s\MLS.Agent\CommandLine\CommandLineParser.cs:line 444
   --- End of inner exception stack trace ---
   at System.RuntimeMethodHandle.InvokeMethod(Object target, Object[] arguments, Signature sig, Boolean constructor, Boolean wrapExceptions)
   at System.Reflection.RuntimeMethodInfo.Invoke(Object obj, BindingFlags invokeAttr, Binder binder, Object[] parameters, CultureInfo culture)
   at System.Delegate.DynamicInvokeImpl(Object[] args)
   at System.Delegate.DynamicInvoke(Object[] args)
   at System.CommandLine.Invocation.ModelBindingCommandHandler.InvokeAsync(InvocationContext context)
   at System.CommandLine.Invocation.InvocationPipeline.<>c__DisplayClass2_0.<<InvokeAsync>b__0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at MLS.Agent.CommandLine.CommandLineParser.<>c.<<Create>b__7_3>d.MoveNext() in F:\workspace.13\_work\1\s\MLS.Agent\CommandLine\CommandLineParser.cs:line 131
--- End of stack trace from previous location where exception was thrown ---
   at System.CommandLine.Invocation.InvocationExtensions.<>c.<<UseParseErrorReporting>b__16_0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.CommandLine.Invocation.InvocationExtensions.<>c__DisplayClass8_0.<<UseTypoCorrections>b__0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.CommandLine.Invocation.InvocationExtensions.<>c.<<UseSuggestDirective>b__7_0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.CommandLine.Invocation.InvocationExtensions.<>c.<<UseParseDirective>b__6_0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.CommandLine.Invocation.InvocationExtensions.<>c.<<UseHelp>b__14_0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.CommandLine.Builder.CommandLineBuilderExtensions.<>c__DisplayClass3_0.<<UseVersionOption>b__0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.CommandLine.Invocation.InvocationExtensions.<>c.<<RegisterWithDotnetSuggest>b__17_0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.CommandLine.Invocation.InvocationExtensions.<>c__DisplayClass5_0.<<UseExceptionHandler>b__0>d.MoveNext()
```

**Comment by akshita31** [comment]
This is a duplicate of https://github.com/dotnet/try/issues/299

**Comment by akshita31** [comment]
This is fixed in #380
